assets/store/Viewport.js

```diff
@@ -109,6 +110,17 @@ export default class Viewport extends Reactive {
     if (typeof value == 'boolean') value = value ? 'yes' : 'no';
     el.content = value;
   }
+
+  update(params) {
+    var body = document.getElementsByTagName('body')[0];
+    if (this.compactDisplay) {
+      body.classList.add('is-compact');
+    }
+    else {
+      body.classList.remove('is-compact');
+    }
```

Comment:
This can be golfed into `body.classList[this.compactDisplay ? 'add' : 'remove']('is-compact');`

Reply:
Done.